Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -1288,9 +1288,6 @@ void CSystem::CreateSystemVars()
         " 0 = on PC if vsync is off auto throttles fps while in menu or game is paused (default)\n"
         "-1 = off");
 
-    REGISTER_CVAR2("sys_maxTimeStepForMovieSystem", &g_cvars.sys_maxTimeStepForMovieSystem, 0.1f, VF_NULL, "Caps the time step for the movie system so that a cut-scene won't be jumped in the case of an extreme stall.");
```

Comment:
I don't think you meant to remove this Console Variable registration.
The sys_maxTimeStepForMovieSystem isn't duplicated in the AZ Console, so this should remain.

Reply:
no i did not meant to do that but  i corrected it